projects/kit/components/multi-select/multi-select.component.ts

```diff
@@ -118,6 +118,9 @@ export class TuiMultiSelectComponent<T>
     @Input()
     rows = Infinity;
 
+    @Input()
+    tagAutoColor = false;
```

Comment:
sorry, I meant renaming for `multi-select` too :) so it will be `autoColor` everywhere

Reply:
Ok np :)